packages/redux/src/bags/selectors.ts

```diff
@@ -711,3 +711,27 @@ export const isBagOperationLoading = (
   state: StoreState,
   bagOperationId: BagOperation['id'],
 ) => getIsBagOperationLoading(state.bag as BagsState)[bagOperationId];
+
+/**
+ * Retrieves current promocodes information in bag.
+ *
+ * @param state - Application state.
+ *
+ * @returns - Bag PromoCodes Information, `undefined` if there are no bag id
+ *
+ * @example
+ * ```
+ * import { getBagPromoCodesInformation } from '@farfetch/blackout-core/bags/redux';
+ *
+ * const mapStateToProps = state => ({
+ *     bagPromoCodesInformation: getBagPromoCodesInformation(state)
+ * });
+ * ```
+ */
+export const getBagPromoCodesInformation = (state: StoreState) => {
+  const bagId = getBagId(state);
+  if (!bagId) {
```

Comment:
Add some line spacing here to make it easier to read, please.

Reply:
done